src/components/ConnectWallet/Wallets/ElectrumAndroid.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <card-widget
+    header="Here's how to connect Electrum Wallet (Android) to your Umbrel"
+  >
+    <div class="px-3 px-lg-4 pb-3">
+      <ol class="connect-wallet-steps">
```

Comment:
With `ol`, we don't need to manually number the steps! It does so on its own, so less chance of error on our end.